mitodl/ocw-hugo-projects
# chore(config) add pre-commit-config

### What are the relevant tickets?
N/A

### Description (What does it do?)

Adds pre-commit config to make the dev experience more coherent.


### How can this be tested?
1. Clone this repo and navigate into it.
2. Switch to the branch `hussaintaj/config-pre-commit`.
3. Run
    ```
    pre-commit install
    ```
4. Mess up some YAML files.
5. Run
    ```
    git add . && pre-commit run
    ```
6. Expect pre-commit to run yamlint or prettier checks and show an error (if you messed up the files correctly).
